docs/en/admins/10_ServerConfig.md

```diff
@@ -91,7 +91,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required (fixes #3076).
```

Comment:
If you think the addition between parentheses is useful I'd rephrase it as `see #3076`. Fixes strikes me as rather an odd thing to see in a sample config.

Reply:
Done